ui/app/routes/vault/cluster/metrics/index.js

```diff
@@ -1,25 +1,26 @@
 import Route from '@ember/routing/route';
 import ClusterRoute from 'vault/mixins/cluster-route';
 import { hash } from 'rsvp';
-import { format, addDays } from 'date-fns';
+import { getTime } from 'date-fns';
 import { parseDateString } from 'vault/helpers/parse-date-string';
 
 const getActivityParams = ({ start, end }) => {
-  // Expects MM-YYYY format
+  // Expects MM-yyyy format
   // TODO: minStart, maxEnd
   let params = {};
   if (start) {
     let startDate = parseDateString(start);
     if (startDate) {
       // TODO: Replace with formatRFC3339 when date-fns is updated
-      params.start_time = format(addDays(startDate, 1), 'X');
+      // converts to milliseconds, divide by 1000 to get epoch
+      params.start_time = getTime(startDate) / 1000;
```

Comment:
Pretty sure we added days because there was some weirdness with UTC, but if the tests pass with this update we should be ok